include/image-commands.mk

```diff
@@ -246,7 +246,7 @@ define Build/elx-header
 endef
 
 define Build/eva-image
-	$(STAGING_DIR_HOST)/bin/lzma2eva $(KERNEL_LOADADDR) $(KERNEL_LOADADDR) $@ $@.new
+	$(STAGING_DIR_HOST)/bin/lzma2eva $(KERNEL_LOADADDR) $(KERNEL_ENTRY) $@ $@.new
```

Comment:
Wasn't there a dedicated PR for this one? Please drop it here. It isn't related to the subject of change and looking at the already support AVM boards not strictly required.

Reply:
The other PR seems not to be accepted, but I will remove it here.